charts/seed-monitoring/charts/grafana/dashboards/owners/apiserver-overview.json

```
@@ -2122,9 +2122,9 @@
       "steppedLine": false,
       "targets": [
         {
-          "expr": "sum(etcd_db_total_size_in_bytes{job=~\"$apiserver\"})",
+          "expr": "topk(1, etcd_db_total_size_in_bytes{job=~\"$apiserver\"}) by (endpoint)",
```

```suggestion
          "expr": "max(etcd_db_total_size_in_bytes{job=~\"$apiserver\"})",
```

`max` (or `min`) will also reduce the timeseries to a single value.

I'm not sure if we need the endpoint in the legend